Devscord.DiscordFramework.UnitTests/Commands/RunnerOfIBotCommandMethodsTests/GetterOfThingsAboutBotCommand.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Devscord.DiscordFramework.Framework.Commands.Builders;
+using Devscord.DiscordFramework.Framework.Commands.Services;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Moq;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+using Devscord.DiscordFramework.Framework.Commands;
+using Devscord.DiscordFramework.Framework.Architecture.Controllers;
+
+namespace Devscord.DiscordFramework.UnitTests.Commands.RunnerOfIBotCommandMethodsTests
+{
+    class GetterOfThingsAboutBotCommand
```

Comment:
https://pl.wikipedia.org/wiki/Test_jednostkowy

Reply:
rozumiem, że mam usunąć tą klasę i wszystko po kolei robić w poszczególnych testach?

Reply:
mam wrażenie że fakt że taką klase musiałeś zrobić wynika z tego, że testy same w sobie są zaprojektowane

testy jednostkowe powinny sprawdzać pojedyncze drobne funkcjonalności odizolowane od reszty 
a tutaj mamy bardzo dużą ilość kodu, przez co testy są zależne od wielu elementów i sprawdzają wiele funkcjonalności

celem jest napisanie tego tak, żeby kodu w testach było jak najmniej i był jak naprostszy

w testach też mogą być błędy - im mniej kodu tym mniej szans że tam będą
no i też im mniej kodu tym prościej zauważyć ewentualne błędy 
jak mamy wielu klas z wieloma zależnościami, to często bardzo trudno zauważyć że coś jest nie tak